explainer.md

```diff
@@ -88,6 +89,21 @@ Items that are designed to be purchased once and last permanently in the user’
 itemService.acknowledge(purchaseToken, 'onetime');
 ```
 
+### Checking existing purchases
+
+The `listPurchases` method allows a client to get a list of items that are currently owned or purchased by the user. This may be necessary to check for entitlements (eg. whether a subscription, promotional code, or permanent upgrade is active) or to recover from network interruptions during a purchase (e.g. item is purchased but not yet acknowledged).
+
+
+```js
+purchases = await itemService.listPurchases();
+for (p in purchases) {
+  if (p.acknowledged)
```

Comment:
Below this is called `isAcknowledged`.

Reply:
Is there a convention to use / not use "isX" for booleans? From a quick search I see plenty of examples of both in IDL files. I'll go with the shorter one.

Reply:
Good call -- I think the local consistency is important so for that I looked at Payment Request API ([IDL](https://w3c.github.io/payment-request/#idl-index)).

It has a number of booleans that do not have an "is" prefix... `PaymentItem.pending` and `PaymentShippingOption.selected` in particular.

So Ack.